### DISCUSSION TITLE
PendingTimeout in 2nd phase

### DISCUSSION THREAD

**Discussion by billgets123** [PaddlePaddle/PaddleSpatial#219] [BaiduKDDCup2022]
Hello,

When submitting to the 2nd phase test data I get the following error:

![微信图片_20220623182750](https://user-images.githubusercontent.com/18434486/175282157-5eeb6069-3bf7-47d3-9dee-e34fad3f524e.png)

However, this model takes about only 1 hour to finish in the 1st phase, I wonder why it failed in the 2nd phase. Our team name is hahaha and the submit time is 2022-06-23 16:15.

**Comment by weicao1990** [comment]
It seems that the time limit of phase 2 is 2 hours instead of 10 hours in phase 1. Is that correct?

**Comment by hachinoone** [comment]
Could you please tell me how did you solve this problem? I have also met this error twice. Thank you.

**Reply by billgets123** [reply]
I haven't solve this problem by far, but it's weird that other similar models of mine can be submitted successfully.

**Reply by hachinoone** [reply]
> I haven't solve this problem by far, but it's weird that other similar models of mine can be submitted successfully.

Yes. I have also sucessfully submitted another similar model. Thank you.

**Comment by btrotta** [comment]
I am also experiencing the same problem: a submission I made in Phase 1 times out in Phase 2. Moreover, this is a very simple baseline submission that just predicts the pre-calculated hourly whole-farm average generation for each point, so it should run very quickly. Team name: BT. Submit time: 2022-06-24 16:39.

**Comment by shaido987** [comment]
I also just encountered this issue.

Did any of you succeed in resubmitting the same (and not just similar) model? Maybe it could be related to the server load at the time of submission?
